include/class.avatar.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/*********************************************************************
+    class.avatar.php
+
+    Avatar sources for users and agents
+
+    Jared Hancock <jared@osticket.com>
+    Peter Rotich <peter@osticket.com>
+    Copyright (c)  2006-2015 osTicket
+    http://www.osticket.com
+
+    Released under the GNU General Public License WITHOUT ANY WARRANTY.
+    See LICENSE.TXT for details.
+
+    vim: expandtab sw=4 ts=4 sts=4:
+**********************************************************************/
+
+abstract class Avatar {
+    var $user;
+
+    function __construct($user) {
+        $this->user = $user;
+    }
+
+    abstract function getUrl($size);
```

Comment:
The interface should add `__toString()` since the system coerces them to strings